scripts/pi-hole/js/footer.js

```diff
@@ -226,15 +225,45 @@ function initCheckboxRadioStyle() {
   }
 }
 
-var systemTimer, sensorsTimer, versionTimer;
+var systemTimer, versionTimer;
 function updateInfo() {
   updateSystemInfo();
-  updateSensorsInfo();
   updateVersionInfo();
   updateFtlInfo();
   checkBlocking();
 }
 
+function updateQueryFrequency(intl, frequency) {
+  let freq = parseFloat(frequency) * 60;
+  let unit = "q/min";
+  let title = "Queries per minute";
+  if (freq > 100) {
+    freq /= 60;
+    unit = "q/s";
+    title = "Queries per second";
+  }
+
+  // Determine number of fraction digits based on the frequency
+  // - 0 fraction digits for frequencies > 10
+  // - 1 fraction digit for frequencies between 1 and 10
+  // - 2 fraction digits for frequencies < 1
```

Comment:
I still think we should only go with full integer values and "<1" only.

Reply:
I think showing decimal places only for small numbers (`< 10`) is fine. 
We already show decimal places for the CPU and MEM.

Reply:
Are we making any progress on this one? I still think low numbers deserve some decimal places as to not artificially loose information when it is there - even on low query count

Reply:
I agree with DL6ER. We could show the decimal places for small numbers.

Reply:
Ok, then proceed with the current implementation. Time and users will tell if we need to change :-)